src/pieces/autocommit/autocommit.py

```diff
@@ -12,6 +12,9 @@
 from pieces_os_client.models.qgpt_relevance_input import QGPTRelevanceInput
 from pieces_os_client.models.qgpt_relevance_input_options import QGPTRelevanceInputOptions
 
+accepted_paths = ["csx","html""htm","css","dart","java","js","jsx","json","php","php3","php4",
+                  "php5","php7","php8","py","py3","curl","go","swift","rs","ts","tsx","rb","bat",
+                  "lua","md","bash","r","sql","groovy","kt","vue"]
```

Comment:
is this the "language" support? if so I would add this in to its own file so that we can use it in the future

Also how are we using this??

if we are checking to see if our extension is in here then we will 100% want to use a Set or a dictionary over an array because of look up speed

Reply:
ahh i see down below, let modify this to be a map, and lets more to a new file because this will be used in other places in the future as well